db/schema.rb

```diff
@@ -1938,6 +1938,18 @@
   add_index "portal_nces06_schools", ["STID"], :name => "index_portal_nces06_schools_on_STID"
   add_index "portal_nces06_schools", ["nces_district_id"], :name => "index_portal_nces06_schools_on_nces_district_id"
 
+  create_table "portal_offering_embeddable_metadata", :force => true do |t|
+    t.integer  "offering_id"
+    t.integer  "embeddable_id"
+    t.string   "embeddable_type"
+    t.boolean  "enable_score",    :default => false
+    t.integer  "max_score"
+    t.datetime "created_at",                         :null => false
+    t.datetime "updated_at",                         :null => false
+  end
+
+  add_index "portal_offering_embeddable_metadata", ["offering_id", "embeddable_id", "embeddable_type"], :name => "index_portal_offering_metadata", :unique => true
+
```

Comment:
This schema file appears to include the tables from the reports branch. Can you fix that? I think you'll need to checkout the reports branch and then roll back its migrations, then switch back to this branch.